### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by timoludwig**
### Short description

Update dependencies

### Proposed changes

- Update npm depdendencies
- Update pip depdendencies

### Side effects

I don't really understand why djlint has such a huge diff again, but I don't see an immediate problem with it (apart from a few annoying git conflicts).
However, we could also try to identify which version bump caused these changes to then clarify whether this is a bug of djlint (which might get reverted in future versions) or a new feature / style guideline which will stay.

### Resolved issues

- https://github.com/digitalfabrik/integreat-cms/security/dependabot/32

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

**Review by ulliholtgrave — APPROVED** [review]
Lgtm :)

File: pyproject.toml [hunk 1]
```diff
@@ -121,66 +121,68 @@ pinned = [
...
     "multidict==6.0.4",
-    "natsort==8.3.1",
+    "natsort==8.4.0",
     "oscrypto==1.3.0",
     "parso==0.8.3",
     "pexpect==4.8.0",
     "pickleshare==0.7.5",
-    "Pillow==9.5.0",
-    "prompt-toolkit==3.0.38",
+    "Pillow==10.0.0",
```
Comment (timoludwig) [inline pyproject.toml:161]: The update of the pillow library seems to break the media upload:
```
  File "/home/integreat/integreat-cms/integreat_cms/cms/utils/media_utils.py", line 53, in generate_thumbnail
    image.thumbnail((size, size), resample=Image.ANTIALIAS)
                                           ^^^^^^^^^^^^^^^
AttributeError: module 'PIL.Image' has no attribute 'ANTIALIAS'
```

This is another example why #1997 is needed :sweat_smile:

Comment (timoludwig) [reply pyproject.toml:161]: Ok, this was an easy fix :laughing: 
https://stackoverflow.com/questions/76616042/attributeerror-module-pil-image-has-no-attribute-antialias

**Review by MizukiTemma — APPROVED** [review]
Now media library works too. Thank you 👍

**Pull request merged**